### PR TITLE
Dapp intercept transaction

### PR DESCRIPTION
**Pull request by xiaoluSF**
收藏先暂往后放一放

### CODE REVIEW

**Comment by xiaoluSF** [comment]
@ashchan `ETHMessageSignerTests` 测试数据是安卓帮忙签的，我这边通过了云斗龙的验证但是数据显示跟安卓不一致，明早再跟段总研究一下。

**Comment by xiaoluSF** [comment]
> It seems the local block (closure) is unnecessary. Isn't a simple let and if sufficient like this:
> 
> ```
> let script: String
> if error != nil {
>     script = "..."
> } else {
>     script = "..."
> }
> ```

OK , next commit will fix it.

**Comment by ashchan** [comment]
> @ashchan `ETHMessageSignerTests` 测试数据是安卓帮忙签的，我这边通过了云斗龙的验证但是数据显示跟安卓不一致，明早再跟段总研究一下。

ok

File: Neuron/Sections/Dapp/DappViewController.swift [hunk 1]
```diff
@@ -29,8 +29,7 @@ class DappViewController: UIViewController, WKUIDelegate, WKNavigationDelegate,
 
     func didAddSubLayout() {
         webView = WKWebView.init(frame: CGRect(x: 0, y: 20, width: ScreenSize.width, height: ScreenSize.height - 49 - 20))
-//        let url = URL(string: "http://dapp.cryptape.com")
-        let url = URL(string: "http://192.168.10.100:3000")
+        let url = URL(string: "http://dapp.cryptape.com")
```
Comment (ashchan) [inline Neuron/Sections/Dapp/DappViewController.swift]: Should use https.

File: Neuron/Sections/Dapp/DappViewController.swift [hunk 2]
```diff
@@ -29,8 +29,7 @@ class DappViewController: UIViewController, WKUIDelegate, WKNavigationDelegate,
 
     func didAddSubLayout() {
         webView = WKWebView.init(frame: CGRect(x: 0, y: 20, width: ScreenSize.width, height: ScreenSize.height - 49 - 20))
```
Comment (ashchan) [inline Neuron/Sections/Dapp/DappViewController.swift]: * `init` is unnecessary.
* Hardcoding `20` (status bar height?) and `49` is bad smell and that won't adapt iPhone X screen. (But let's leave this as and discuss in next UI session)

**Pull request merged**